graphqlapi/src/main/java/io/stargate/graphql/schema/cqlfirst/dml/fetchers/QueryFetcher.java

```diff
@@ -93,16 +97,63 @@ private BoundQuery buildQuery(DataFetchingEnvironment environment, DataStore dat
         limit = (int) limitObj;
       }
     }
-    return dataStore
-        .queryBuilder()
-        .select()
-        .column(buildQueryColumns(environment))
-        .from(table.keyspace(), table.name())
-        .where(buildClause(table, environment))
-        .limit(limit)
-        .orderBy(buildOrderBy(environment))
-        .build()
-        .bind();
+    QueryBuilder.QueryBuilder__41 queryBuilder =
+        dataStore
+            .queryBuilder()
+            .select()
+            .column(buildQueryColumns(environment))
+            .from(table.keyspace(), table.name())
+            .where(buildClause(table, environment))
+            .limit(limit)
+            .orderBy(buildOrderBy(environment));
+
+    addAggregationFunctions(environment, queryBuilder);
```

Comment:
I think that we can leverage an existing `QueryFetcher` without the need for a new `AggregatedFunctionsFetcher`, @olim7t wdyt?